src/System.Management.Automation/namespaces/FileSystemContentStream.cs

```diff
@@ -733,7 +733,7 @@ private bool ReadDelimited(bool waitChanges, ArrayList blocks, bool readBackward
                 //    (Trimming it during backward reading would not only be unnecessary, but could interfere with determining the correct start position.)
                 string contentString = content.ToString();
                 blocks.Add(
-                    !readBackward && contentString.EndsWith(actualDelimiter, StringComparison.Ordinal)
+                    !readBackward && contentString.EndsWith(actualDelimiter, StringComparison.Ordinal) && !_isRawStream
```

Comment:
This looks good;
but original issue says that "Note that Windows Powershell is not affected".
Any ideas why this does not repro in Windows PS?

Reply:
This is because the code is introduced by core ps code change.